mbed-net-sockets/Socket.h

```diff
@@ -1,32 +1,22 @@
 /*
- * PackageLicenseDeclared: Apache-2.0
- * Copyright (c) 2015 ARM Limited
+ * Copyright (c) 2015, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
 #ifndef __MBED_NET_SOCKETS_SOCKET_H__
 #define __MBED_NET_SOCKETS_SOCKET_H__
-
-/* Include all versions of the API */
-#include "v0/Socket.h"
```

Comment:
It's just moved to another place.  Look at the commits in the PR in order and it makes more sense.